### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by Fx-Colliot**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById("root")
 );
 
 // If you want to start measuring performance in your app, pass a function
```
Comment (Thoms-C) [inline src/index.js]: We should delete line 14 (not usefull)

**Comment by LoicBrassart** [comment]
@Thoms-C As long as your feedbacks are not taken into consideration, you shouldn't involve me

**Review by Thoms-C — APPROVED** [review]
ok!

File: package.json [hunk 2]
```diff
@@ -6,7 +6,14 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.3",
     "@testing-library/user-event": "^12.6.2",
+    "axios": "^0.21.1",
+    "cors": "^2.8.5",
```
Comment (LoicBrassart) [inline package.json:10]: this package shouldn't be here

File: package.json [hunk 3]
```diff
@@ -6,7 +6,14 @@
...
+    "dotenv": "^8.2.0",
+    "jwt-decode": "^3.1.2",
+    "proxy": "^1.0.2",
+    "ra-data-simple-rest": "^3.10.4",
```
Comment (LoicBrassart) [inline package.json:14]: I don't see why you need these

**Review by LoicBrassart — APPROVED** [review]
SnE, except for unnecessary packages

**Pull request merged**